package.json

```diff
@@ -79,7 +80,7 @@
     "test_async_stack": "node build/test/test/source/async-stack.js",
     "test_buf": "npx ava --timeout=3m --verbose --concurrency=10 build/test/test/source/buf.js",
     "test_ci_chrome_consumer_live_gmail": "npx ava --timeout=30m --verbose --concurrency=1 build/test/test/source/test.js -- CONSUMER-LIVE-GMAIL STANDARD-GROUP",
-    "test_ci_chrome_consumer": "npx ava --timeout=30m --verbose --concurrency=10 build/test/test/source/test.js -- CONSUMER-MOCK STANDARD-GROUP",
+    "test_ci_chrome_consumer": "npx ava --timeout=30m --verbose --concurrency=10 build/test/test/source/test.js -- CONSUMER-MOCK STANDARD-GROUP  --tap | tap-xunit > report.xml",
```

Comment:
@sosnovsky, hmmm... I'm currently investigating deeply why the --tap parameter isn't functioning.
Do you have any suggestions as to why this may be the case?
Your suggestion would be greatly appreciated.

Reply:
@ioanmo226 it worked for me when I moved `--tap` argument, like this:
`npx ava --timeout=30m --verbose --tap --concurrency=10 ...`

Reply:
Yes, that works!
I also attempted to move the `--tap` argument while including `| tap-xunit > report.xml`. That's why it failed on my end (tests all failed with this one).
Now, I might be able to generate an XML file from the TAP output and then send it to SemaphoreCI.

Thank you for your help.